composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "laravel/installer": "^1.3"
```

Comment:
I just ran `composer require laravel/installer` and that is the current version it grabbed, 1.3.7 is the latest tag on the repo... https://github.com/laravel/installer.
Am I missing something?